src/makeCalculator.js

```diff
@@ -4,7 +4,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add: (num) => (result += num),
+
+    subtract: (num) => (result -= num),
+
+    multiply: (num) => (result *= num),
+
+    divide: (num) => (result /= num),
+
+    operate: function (callback, num) {
+      callback(num);
+
+      return this;
+    },
+    reset: function () {
```

Comment:
Similar to the 'operate' method, you can use the method shorthand syntax for 'reset'. Define it as 'reset() { ... }' to follow the checklist guideline.